src/lxc/seccomp.c

```diff
@@ -1243,5 +1271,57 @@ void lxc_seccomp_free(struct lxc_conf *conf)
 		seccomp_release(conf->seccomp_ctx);
 		conf->seccomp_ctx = NULL;
 	}
+	close_prot_errno_disarm(conf->seccomp_notify_fd);
+	close_prot_errno_disarm(conf->seccomp_notify_proxy_fd);
+	seccomp_notif_free(conf->seccomp_notify_req, conf->seccomp_notify_resp);
+	conf->seccomp_notify_req = NULL;
+	conf->seccomp_notify_resp = NULL;
+#endif
+}
+
+int seccomp_notify_handler(int fd, uint32_t events, void *data,
+			   struct lxc_epoll_descr *descr)
+{
+
+#if HAVE_SCMP_FILTER_CTX
+	int ret;
+	struct lxc_handler *hdlr = data;
+	struct lxc_conf *conf = hdlr->conf;
+	struct seccomp_notif *req = conf->seccomp_notify_req;
+	struct seccomp_notif_resp *resp = conf->seccomp_notify_resp;
+	int listener_proxy_fd = conf->seccomp_notify_proxy_fd;
+	struct seccomp_notify_proxy_msg msg;
+
+	if (listener_proxy_fd < 0)
+		return minus_one_set_errno(EINVAL);
+
+	ret = seccomp_notif_receive(fd, req);
+	if (ret)
+		return minus_one_set_errno(-ret);
+
+	if (req->data.nr != SCMP_SYS(mknod))
```

Comment:
I guess you probably want to drop this, right?

Reply:
Yes, for sure. That is residual from testing.

Reply:
done